omnibus/files/private-chef-cookbooks/private-chef/recipes/config.rb

```diff
@@ -51,6 +51,7 @@
 
   if chef_server_rb_exists
     chef_server_rb_not_empty = !File.zero?(chef_server_path)
+    FileUtils.chmod('o-rwx', chef_server_path)
```

Comment:
I think we probably want to use a `file` resource.  It would probably also be a bit better to do the permissions fixup furhter down after all the symlinking and special handling is over.